apps/ui/src/components/TransactionsListItem.vue

```diff
@@ -8,7 +8,7 @@ const props = defineProps<{ tx: Transaction }>();
 
 const title = computed(() => {
   if (props.tx._type === 'sendToken') {
-    return `Send <b>${_n(formatUnits(props.tx._form.amount, props.tx._form.token.decimals))}</b> ${
+    return `Send <b>${_n(formatUnits(props.tx._form.amount, props.tx._form.token.decimals), 'standard', { formatDust: true })}</b> ${
```

Comment:
This is a slight improvement over the previous version.
Previously, it just showed `0` for small number, now it should show `~0`.